ws2812/ws2812_avr_digispark.go

```diff
@@ -0,0 +1,50 @@
+// +build digispark
+
+package ws2812
+
+// This file implements the WS2812 protocol for 16MHz AVR microcontrollers.
```

Comment:
I think this should have some explanation as to why there are two files for seemingly the same functionality. Something along the lines of "this is a slightly different implementation than the one for the atmega to work around a compiler bug".